### PR TITLE
add audit to check if paste is allowed in password inputs

### PR DESCRIPTION
**Pull request by robin-drexler**
Adresses https://github.com/GoogleChrome/lighthouse/issues/2345

## todo 
- [x] add/format violation output
- [ ] proper copy (help text, description)
- [x] tests

### CODE REVIEW

**Comment by paulirish** [comment]
nice nice! this is on point so far. well done. :)

**Comment by robin-drexler** [comment]
Output currently looks like this:

lighthouse_report

Is this any good?

Could anybody help me with the failing build? I can't seem to figure out why it's breaking. Everything looks pretty good, but it still exits with exit code 1. Thanks in advance! :)

**Comment by robin-drexler** [comment]
I just didn't look thoroughly enough, sorry. There was a linting issue which should now be resolved.

**Comment by robin-drexler** [comment]
I signed it!

**Comment by robin-drexler** [comment]
Should I also create a doc page like https://developers.google.com/web/tools/lighthouse/audits/geolocation-on-load ?

If that's the case, can somebody point me to the respective repo? :)

**Comment by wardpeet** [comment]
thanks @robin-drexler! You can find the documentation here https://github.com/google/WebFundamentals/tree/master/src/content/en/tools/lighthouse/audits

cc @ebidel

File: lighthouse-core/config/default.js [hunk 1]
```diff
@@ -133,7 +134,8 @@ module.exports = {
     "dobetterweb/notification-on-start",
     "dobetterweb/script-blocking-first-paint",
     "dobetterweb/uses-http2",
-    "dobetterweb/uses-passive-event-listeners"
+    "dobetterweb/uses-passive-event-listeners",
+    "dobetterweb/password-inputs-can-be-pasted-into"
```
Comment (wardpeet) [inline lighthouse-core/config/default.js]: nit, all these audits are arranged alphabetical, maybe order them here too

File: lighthouse-core/config/default.js [hunk 2]
```diff
@@ -26,6 +26,7 @@ module.exports = {
       "dobetterweb/response-compression",
       "dobetterweb/tags-blocking-first-paint",
       "dobetterweb/websql",
+      "dobetterweb/password-inputs-with-prevented-paste",
```
Comment (wardpeet) [inline lighthouse-core/config/default.js]: nit, all these audits are arranged alphabetical, maybe order them here too

**Comment by robin-drexler** [comment]
I'd like to finish this during the weekend but probably could use some help with copywriting and docs and some hints on what else might need to be done. 

Could anyone chime in? :)

**Comment by paulirish** [comment]
Huge. Thank you!

🥇 🥇 🥇

**Comment by robin-drexler** [comment]
Fantastic, can't wait to see it in a release. :D 
Thanks for all your help @wardpeet @paulirish :) 🎉

**Comment by paulirish** [comment]
all good! sorry for not being timely in our replies.

**Pull request merged**